.semaphore/semaphore.yml

```diff
@@ -474,7 +474,7 @@ blocks:
       commands:
       - make ci
 
-- name: "cni-plugin"
+- name: "cni-plugin CI"
```

Comment:
I suggest leaving this out since it would make this job inconsistent with the other CI jobs. Or, change all the other CI jobs to have the `CI` suffix. WDYT?

Reply:
agree - just revert this change.